WordPress/src/main/java/org/wordpress/android/ui/SlidingRelativeLayout.java

```diff
@@ -0,0 +1,48 @@
+package org.wordpress.android.ui;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.ViewTreeObserver;
+import android.widget.RelativeLayout;
+
+public class SlidingRelativeLayout extends RelativeLayout {
+
+    private float xFraction = 0;
```

Comment:
Better rename this into something like `mFractionX` to follow the naming conventions. See https://source.android.com/source/code-style.html#follow-field-naming-conventions. Cheers!